frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelRegistry.cy.ts

```
@@ -34,110 +37,128 @@ type HandlersProps = {
-        },
-        'Fraud detection': {
-          metadataType: ModelRegistryMetadataType.STRING,
-          string_value: '',
-        },
-        'Long label data to be truncated abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc abc':
-          {
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const createInterceptConfig = (initial?: HandlersProps) =>
+  ({
+    ...initial,
+  } satisfies HandlersProps);
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)